test/server/static-directory-option.test.js

```diff
@@ -204,111 +208,37 @@ describe('contentBase option', () => {
     });
   });
 
-  describe('to port', () => {
-    beforeAll((done) => {
-      server = testServer.start(
-        config,
-        {
-          contentBase: 9099999,
-          port,
-        },
-        done
-      );
-      req = request(server.app);
-    });
-
-    afterAll((done) => {
-      testServer.close(() => {
-        done();
-      });
-    });
-
-    it('Request to page', (done) => {
-      req
-        .get('/other.html')
-        .expect('Location', '//localhost:9099999/other.html')
-        .expect(302, done);
-    });
-  });
-
-  describe('to external url', () => {
-    beforeAll((done) => {
-      server = testServer.start(
-        config,
-        {
-          contentBase: 'http://example.com/',
-          port,
-        },
-        done
-      );
-      req = request(server.app);
-    });
-
-    afterAll((done) => {
-      testServer.close(() => {
-        done();
-      });
-    });
-
-    it('Request to page', (done) => {
-      req
-        .get('/foo.html')
-        // TODO: hmm, two slashes seems to be a bug?
-        .expect('Location', 'http://example.com//foo.html')
-        .expect(302, done);
-    });
-
-    it('Request to page with search params', (done) => {
-      req
-        .get('/foo.html?space=ship')
-        // TODO: hmm, two slashes seems to be a bug?
-        .expect('Location', 'http://example.com//foo.html?space=ship')
-        .expect(302, done);
-    });
-  });
-
   describe('testing single & multiple external paths', () => {
     afterEach((done) => {
       testServer.close(() => {
         done();
       });
     });
-    it('Should throw exception (string)', (done) => {
-      try {
-        // eslint-disable-next-line no-unused-vars
-        server = testServer.start(config, {
-          contentBase: 'https://example.com/',
-          watchContentBase: true,
-        });
-
-        expect(true).toBe(false);
-      } catch (e) {
-        expect(e.message).toBe('Watching remote files is not supported.');
-        done();
-      }
-    });
-    it('Should throw exception (number)', (done) => {
+    it('Should throw exception (external url)', (done) => {
```

Comment:
@evilebottnawi Is it correct that I should be throwing if `isAbsoluteUrl` for the static option is true?

Reply:
No sure, I think we should improve it on `schema-utils` side, and support only paths, no URLs